MongoDB/recipes/default.rb

```diff
@@ -0,0 +1,33 @@
+#
+# Cookbook:: MongoDB
+# Recipe:: default
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+#'MongoDB::importpkey'
+execute 'import public key cmd' do
+    command "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10"
+    action :run
+    not_if 'apt-key list | grep 7F0CEB10'
+end
+
+#'MongoDB::createfile'
+execute 'create list of files' do
+   command 'echo "deb http://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list'
+   action :run
+   not_if { File.exist?('/etc/apt/sources.list.d/mongodb-org-3.0.list') }
+end
+
```

Comment:
Instead of using these resources "execute 'import public key cmd'" and "execute 'create list of files'" 
use "apt_repository" resource.
Help docs are here @https://docs.chef.io/resource_apt_repository.html